tests/filecheck/backend/riscv_reduce_register_pressure.mlir

```diff
@@ -0,0 +1,12 @@
+// RUN: xdsl-opt -p riscv-reduce-register-pressure %s | filecheck %s
+
+builtin.module {
+  %0 = riscv.li 0 : () -> !riscv.reg<>
+  %1 = riscv.li 0 : () -> !riscv.reg<a0>
+}
```

Comment:
I see. I feel this could use some documentation. Just 1-2 sentences stating that a li0 is replaced to a load from a zero register.